### ISSUE TITLE
installation instructions updates

### ISSUE THREAD

**Issue by kieranjol** [exponential-decay/demystify#89]
I had to install demysitify in a very restrictive network environment, which then made me realise that it would be a bit similar to a non-networked forensic workstation. I had to do some extra steps by manually downloading the two submodules add copying the contents into the relevant folder. 
Do you see any merit in adding these instructions, or is it too much of an edge case? I could see this tool being used in an offline forensic workstation, so having offline instructions would be helpful. I'm happy to update the docs if necessary.

**Comment by ross-spencer** [comment]
> Do you see any merit in adding these instructions, or is it too much of an edge case? I could see this tool being used in an offline forensic workstation, so having offline instructions would be helpful. I'm happy to update the docs if necessary.

Absolutely! Your example is perfect and part of the original conception.

I had lost track of the impact of the submodules in this case. I'm working on packaging for Py3 at the minute, but PIP isn't an option for everyone either so there's something to be done here around releases and making sure everything is included.

If you have a look at the docs it'll be a great help and I'll get that reviewed and merge. And of course, if you can't find time, I'll happily take a look. I think I'll be doing more around this at the weekend.

**Comment by ross-spencer** [comment]
A quick question - would the package in the releases have worked for you? There are some submodule instructions there too: 

* https://github.com/exponential-decay/demystify/releases/tag/v1.0.0

**Comment by kieranjol** [comment]
I used the tar and it worked perfectly! One issue though is that like this issue: https://github.com/APTrust/dart/issues/229 , 7-zip can't open the tar as it has that PAXHEADER issue. I just used the tar command line in windows and did
`tar -xvzf path/to/tar -C path/to/output`

and all was well. perhaps some warning could be added to a release note?
